lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -458,7 +459,7 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
                 (Link.getWallet @'Shelley wa) Default Empty
             expectField
                 (#balance . #available)
-                (`shouldBe` Quantity (initialAmt - feeMax - amt)) ra2
+                (`shouldBe` Quantity (initialAmt - fee - amt)) ra2
```

Comment:
I welcome using the real fee (precise) where before a fee estimation (imprecise) was used. Lets remove `feeMax` and related expectations from this test completely, as its not what should be tested here.